Xamarin.Forms.Platform.UAP/RefreshViewRenderer.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System.ComponentModel;
+using Microsoft.UI.Xaml.Controls;
+using Windows.Foundation;
+using Windows.UI.Xaml.Media;
+using Xamarin.Forms.PlatformConfiguration.WindowsSpecific;
+using Specifics = Xamarin.Forms.PlatformConfiguration.WindowsSpecific.RefreshView;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public class RefreshViewRenderer : ViewRenderer<RefreshView, RefreshContainer>
+	{
+		bool _isDisposed;
+		Deferral _refreshCompletionDeferral;
+
+		public RefreshViewRenderer()
+		{
+			AutoPackage = false;
+		}
+
+		protected override void Dispose(bool disposing)
+		{
+			if (_isDisposed)
+				return;
+
+			if (Control != null && disposing)
+			{
+				Control.RefreshRequested -= OnRefresh;
+
+				if (_refreshCompletionDeferral != null)
+				{
+					_refreshCompletionDeferral.Complete();
+					_refreshCompletionDeferral.Dispose();
+					_refreshCompletionDeferral = null;
+				}
+			}
+
+			_isDisposed = true;
+
+			base.Dispose(disposing);
+		}
+
+		protected override void OnElementChanged(ElementChangedEventArgs<RefreshView> e)
+		{
+			if (e.NewElement != null)
+			{
+				if (Control == null)
+				{
+					var refreshControl = new RefreshContainer
+					{
+						PullDirection = RefreshPullDirection.TopToBottom,
+						Visualizer = new RefreshVisualizer()
```

Comment:
@jsuarezruiz it looks like when you assign a Visualizer the RefreshRequested event fires twice. 

Do we need to assign a Visualizer here?

Reply:
I see it was so you could update the colors. I added a callback to set those colors once WinUI just initializes it itself which looks to work